usermode/distribution.go

```diff
@@ -71,14 +71,16 @@ func NewGenerator() *Generator {
 
 // Generate fills the out argument with random data. The argument must be a pointer.
 // Currently supported types are int and string. The function returns true if
-// the argument was successfully filled with random data, false otherwise.
-// The seed argument controls the uniqueness distribution of the generated
+// a unique seed was generated for the argument and it successfully filled with random data,
+// false otherwise.
+// If uniq argument is false, the function reuses the seeds and always returns true.
+// The col.Population argument controls the uniqueness distribution of the generated
 // data.
-// The size argument controls the size in bytes of the generated data.
-func (g *Generator) Generate(col *ColumnSpec, out interface{}) bool {
+// The col.Size argument controls the size in bytes of the generated data.
+func (g *Generator) Generate(col *ColumnSpec, out interface{}, uniq bool) bool {
```

Comment:
Is this and the following commits related to the "duration" addition?

Reply:
Indirectly yes - testing the tool with duration (hundred of thousands iterations) shown an issue, when we run out of unique seeds for columns that we do not care whether data generated is unique (non-partition and non-cluster columns). This issue manifested itself with gocql errors like:

```
2018/09/18 08:18:28 gocql: batch statement 7 expected 6 values send got 5
2018/09/18 08:18:29 gocql: batch statement 3 expected 6 values send got 5
```